www/user_page.html

```diff
@@ -150,6 +150,19 @@
             </div>
           </div>
         </li>
+        <li>
+          <div class="item-content user-switch">
+            <div class="item-inner">
+              <div class="item-title label"><small>Notiser två timmar före eventanmälan stänger</small></div>
```

Comment:
I think it looks better if this small tag only affects ios, because the text becomes unnecessarily small on material. So, this should probably be done in css in ios-overrides